src/lib/containers/download_list/DownloadListTable.tsx

```diff
@@ -379,15 +371,16 @@ export default function DownloadListTable(props: DownloadListTableProps) {
                     sortColumn('file')
                   }}
                 >
-                  <FontAwesomeIcon
-                    icon={
-                      sortedColumn.column === 'file'
-                        ? sortedColumn.isDescending === false
-                          ? faSortAmountDown
-                          : faSortAmountUp
-                        : faSortAmountDown
-                    }
-                    color={sortedColumn.column === 'file' ? 'white' : ''}
+                  <IconSvg
```

Comment:
Ok, these don't look great, but this component uses the old Download List services and is deprecated.  It should just be deleted at this point instead of patching!